courses/machine_learning/deepdive2/structured/solutions/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -34,6 +34,22 @@
     "## Set up environment variables and load necessary libraries"
```

Comment:
<p>I had to change this from "from babyweight.trainer import model" to "from trainer import model"</p><p>Otherwise, I got an error about babyweight module not found on CAIP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/914/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>